content/blocks/backlog/index.md

```diff
@@ -10,7 +10,13 @@ vocabulary="Backlog"
 
 +++
 
-You will find your backlog in the [Backlog](../backlog/). This is a list of all the tickets for the module. You can also go week by week by looking in the sprint backlog.
+In software development, we break down complex projects into smaller, manageable parts, which we work on for a week or two. These periods are called "sprints."
+
+A sprint backlog is like a to-do list. It lists what the team has decided to work on this sprint. It's chosen from a larger list, usually called the "product backlog," which holds the entire project to-do list.
+
+For your course, the mandatory work has been scheduled already. You will add any other tickets you want to work on to your backlog and schedule them according to your own goals and capacity. Use your planning board to do this.
+
+You will find your backlog in the [Backlog](../backlog/).
```

Comment:
This link appears to be dead for fundamentals, as there is no top-level backlog page. Unclear to me if that's just an oversight in fundamentals, or deliberate in which case this link may have to made conditional somehow.